bool enabled
float64 speed
float64 steering_angle